### PR TITLE
Added an image viewer for pictures on the roadmap

### PR DESCRIPTION
**Pull request by mmzeeman**
Changes:

 - Adapted mediaclass settings to prevent png files from being transformed to jpg
 - Added three mediaclasses for small, middle, and large images.
 - Added a thumbnail viewer which displays images full page.

### CODE REVIEW

File: lib/css/z.thumbviewer.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    cursor: pointer;
+    z-index: 8000;
+    font-size: xx-large;
+    text-align: right;
+    padding-top: 10px;
+    padding-right: 10px;
+}
+
+span.thumbviewer.overlay:before {
+    content: "⊗"
```
Comment (OldhamMade) [inline lib/css/z.thumbviewer.css:17]: Might be good to specify a font here, for safety.

File: lib/js/modules/z.thumbviewer.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        } else {
+            $(image[0]).on("load", this.init.bind(this));
+        }
+    },
+	
+    init: function() {
+        var image = this.element;
+        var loadImageFun = this.loadImage.bind(this);
+
+        this.bigImg = $('<img/>').hide();
```
Comment (OldhamMade) [inline lib/js/modules/z.thumbviewer.js:34]: Worth wrapping in a `<picture>`, for more styling options in the future?

**Review by OldhamMade — APPROVED** [review]
LGTM, just 2 possible improvements.

File: lib/js/modules/z.thumbviewer.js (hunk 2 above)
Comment (mmzeeman) [reply lib/js/modules/z.thumbviewer.js:34]: Sure. We need something which works for the whole site. This hooks into the media publishing part of zotonic. See: http://docs.zotonic.com/en/latest/developer-guide/media.html. It uses `{% media ... %}` and `{% image ...%}` tags to help with placing media on pages. Normally this places the img, picture, figure tags. New functionality is also coming here (srcset). It is probably a good idea to have a strategy which works across the whole site, and then hook js-viewers like this into it. This is a very simplistic one.

File: lib/css/z.thumbviewer.css (hunk 1 above)
Comment (mmzeeman) [reply lib/css/z.thumbviewer.css:17]: Like this? Helvetica, Arial, Sans-serif?

**Pull request merged**